GettingStarted.md

```diff
@@ -0,0 +1,167 @@
+# Introduction
+Welcome to impress.js! This presentation framework allows you to create stunning presentations with the power of CSS3 transformations.
+**NOTE:** This Guide is not made for you, if you have never written HTML and/or CSS before. Knowing your way around in JavaScript certainly helps, but is not a necessity. You may still continue this tutorial and try to understand what we do as you go. 
+
+For more advanced and complete documentation, you might prefer the [DOCUMENTATION](DOCUMENTATION.md).
+
+# Getting started with impress.js
+## Installation / acquiring the framework
+First of all, you need to know, if you are going to have WiFi connection when you hold your presentation. If you are not sure, please use the method where you download the file instead of the cdn.
+
+### Including from cdn
+Loading the script from the cdn is quite straight forward. If you copy the below example code, you need to do nothing else, impress will be loaded automatically.
+
+**Direct links to different versions of the impress.js file**
+- V2.0.0: https://cdn.jsdelivr.net/gh/impress/impress.js@2.0.0/js/impress.js
+- V1.1.0: https://cdn.jsdelivr.net/gh/impress/impress.js@1.1.0/js/impress.js
+- Source: https://cdn.jsdelivr.net/gh/impress/impress.js/js/impress.js
+
+## Download the file to your PC
+Head to the releases tab and download the source code as zip or as a tarball. Go ahead and unzip / untar it. You need to copy the folder */js/* and the folder */css/* (if you want to make your life a bit easier) into the folder you are working in. 
+
```

Comment:
I still think this is a bit fuzzy. It's not clear that you don't *need* to download /css/* rather that is **entirely** optional.

My suggestion would be to just mention `/js/*` in this section, and then mention `/css/*` only in the section further below that you start to speak about CSS.

Reply:
Was also missing another # there, as this was supposed to be h3 class